docs/fr-FR/README.md

```diff
@@ -1,6 +1,6 @@
 <div align="center">
     <a href="https://github.com/all-contributors/all-contributors">
-        <img src="other/logo-full.png" alt="✨ All Contributors ✨" width="800px" />
+        <img src="../other/logo-full.png" alt="✨ All Contributors ✨" width="800px" />
```

Comment:
the image still is broken in markdown file, mind update again? @Berkmann18

Reply:
Ooh, I'll get on it.